apps/web/src/styles/Proposals.css.ts

```diff
@@ -27,21 +27,27 @@ export const proposalFormTitle = style({
   },
 })
 
-export const submitProposalBtn = style({
+export const delegateBtn = style({
   fontSize: '1rem',
   fontFamily: 'ptRoot!important',
   borderRadius: '12px',
   height: 40,
-  color: 'black',
-  backgroundColor: '#F2F2F2',
-  selectors: {
-    '&:hover': {
-      color: 'white',
-      backgroundColor: '#00A8FF',
-    },
+  color: '#000',
+  backgroundColor: '#FFF',
+  ':hover': {
+    backgroundColor: '#F9F9F9!important',
   },
 })
 
+export const submitProposalBtn = style({
+  fontSize: '1rem',
+  fontFamily: 'ptRoot!important',
```

Comment:
do we need this?

Reply:
The code comparison got kind of weird but those styles are already on submitProposalBtn